src/handlers.ts

```diff
@@ -73,13 +80,14 @@ export class MyHandler extends Handler {
   async doIT() {
     let opt;
     let opt2;
```

Comment:
lint error on these 2 lines. These are never reassigned so should be const. So the separate assignment line is excessive.

Reply:
Have changed it to const.